scratch/stan_chol_exmpl.R

```diff
@@ -0,0 +1,200 @@
+library(tidyverse)
+library(rstan)
+
+
+
```

Comment:
This is where I would add (in comments since its a script) the model youre fitting to!